src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -1,69 +1,76 @@
-import React, { useCallback, useEffect, useState } from "react";
+import React, { useCallback, useEffect, useState, ChangeEvent } from "react";
+import classNames from "classnames";
+
+import { faSearch } from "@fortawesome/free-solid-svg-icons";
 
 import { VenueEvent } from "types/venues";
 import { Room, RoomTypes } from "types/rooms";
 import { User } from "types/User";
 
 import { WithId } from "utils/id";
-import { currentVenueSelectorData, venueEventsSelector } from "utils/selectors";
 import { isTruthy } from "utils/types";
+import { formatToWeekdayFullname, formatUtcSeconds } from "utils/time";
+import { currentVenueSelectorData, venueEventsSelector } from "utils/selectors";
 
 import { useWorldUsers } from "hooks/users";
 import { useSelector } from "hooks/useSelector";
 import { useProfileModalControls } from "hooks/useProfileModalControls";
 
+import { InputField } from "components/atoms/InputField";
 import { RoomModal } from "components/templates/PartyMap/components";
```

Comment:
Order:

- pages
- templates
- organisms
- molecules
- atoms

Basically 'largest' to 'smallest'.

```suggestion
import { RoomModal } from "components/templates/PartyMap/components";

import { InputField } from "components/atoms/InputField";
```

Reply:
thanks for the explanations. Didn't have a clear understanding about these import-orders.
fixed: https://github.com/sparkletown/sparkle/pull/1328/commits/26ca60e7ec8efdead414b276855c0baaadd66777